### ISSUE TITLE
experimental results

### ISSUE THREAD

**Issue by with-twilight** [hongsukchoi/3DCrowdNet_RELEASE#7]
Thank you for sharing.
I tried to reproduce your experimental results, but both results were far from those given in your paper.

First result:
MPJPE from mesh: 493.55 mm
PA MPJPE from mesh: 112.53 mm
MPVPE from mesh: 529.30 mm

Second result:
MPJPE from mesh: 644.32 mm
PA MPJPE from mesh: 103.72 mm
MPVPE from mesh: 661.93 mm

Can you share the hyperparameter setting or give some instructions?
pytorch:1.6,cuda:10.2

**Comment by hongsukchoi** [comment]
Hi!

Did you see the instructions for reproducing?
The result is obviously not related to the hyperparameter. There will be a critical bug.

**Comment by with-twilight** [comment]
Hi,

May I ask which one might be the document you are talking about?

**Comment by hongsukchoi** [comment]
It's indicated at the bottom of README.
Seems many people are missing this. Maybe should I change the 'Running' to 'Reproducing'?

<img width="1022" alt="스크린샷 2022-06-01 22 09 15" src="https://user-images.githubusercontent.com/31090011/171533189-9d2bea17-fe95-448a-b13d-537406c5d386.png">

**Comment by with-twilight** [comment]
I noticed this file, the above results are the results obtained after testing the model trained according to the file.
I forgot to say that the results in Table 5 are not much different.

**Comment by hongsukchoi** [comment]
I don't understand... all hyperparamers are as in yaml files or the config file.

Which tables are you trying to reproduce?

**Comment by with-twilight** [comment]
I would like to reproduce the results of Table 5 and Table 8. 
The results of Table 5:
MPJPE from mesh: 86.37 mm
PA MPJPE from mesh: 55.57 mm
MPVPE from mesh: 108.73 mm

The results of Table 8:
First result:
MPJPE from mesh: 493.55 mm
PA MPJPE from mesh: 112.53 mm
MPVPE from mesh: 529.30 mm

Second result:
MPJPE from mesh: 644.32 mm
PA MPJPE from mesh: 103.72 mm
MPVPE from mesh: 661.93 mm

If the hyperparameters are all given in the config file, is my problem maybe because of the random seed or something else？

**Comment by hongsukchoi** [comment]
errors like that high only occur if there is a critical bug. I think you must have changed code somewhere. Can you send me your code? redarknight@snu.ac.kr

**Comment by with-twilight** [comment]
The E-mail was sent to you.
Thank you very much.
